django/db/migrations/migration.py

```diff
@@ -20,6 +21,10 @@ class Migration:
     Graph as instances, having been initialized with their app label and name.
     """
 
+    RECURSION_ERROR_MESSAGE = (
```

Comment:
Even if it's also used in the `unapply` method?

Reply:
I'd say _unapplying_ is a form of _applying_? My main concern is that is attribute is not bound to any method right now, it could be the message of a recursion error happening during the `mutate_state` method for example.

Reply:
That's a reasonable argument, I'll rename.